routes/api.php

```diff
@@ -20,8 +20,11 @@
 Route::middleware('auth:sanctum')->get('/user', function (Request $request) {
     return $request->user();
 });
+
 Route::apiResource('articles',ArticleController::class);
 
 Route::get('/deleted',[ArticleController::class,'deleted_articles']);
```

Comment:
make the route url full meaning name ( replace the deleted => deleted_articles )